be/src/vec/functions/function_bitmap.cpp

```diff
@@ -382,6 +382,9 @@ class FunctionBitmapAlwaysNull : public IFunction {
             if (check_column<ColumnInt8>(nested_column)) {
                 Impl::template vector<ColumnInt8>(offset_column_data, nested_column,
                                                   nested_null_map, res, null_map);
+            } else if (check_column<ColumnUInt8>(nested_column)) {
```

Comment:
Is UInt16/32/64 also needed?

Reply:
not need, because doris does not support uint....
Here uint8 is boolean type

Reply:
need check datetime it is uint64

Reply:
> need check datetime it is uint64

datetime is casted to int64 if used in bitmap_from_array.